traffic_ops/traffic_ops_golang/user/user.go

```diff
@@ -204,7 +206,17 @@ func (this *TOUser) Read(h http.Header, useIMS bool) ([]interface{}, error, erro
 
 	groupBy := "\n" + `GROUP BY u.id, r.name, t.name`
 	orderBy = groupBy + orderBy
-	query := this.SelectQuery() + where + orderBy + pagination
+
+	version := inf.Version
+	if version == nil {
+		return nil, nil, fmt.Errorf("TOUsers.Read called with invalid API version: %d.%d", version.Major, version.Minor), http.StatusInternalServerError, nil
```

Comment:
Since `version` is nil in here, this line will always segfault trying to dereference `version.Major` and `version.Minor`

Reply:
oops